sympy/stats/tests/test_discrete_rv.py

```diff
@@ -109,7 +140,8 @@ def test_cf(dist, support_lower_limit, support_upper_limit):
 
         # second function is the Fourier transform of the density function
         f = lambdify([x, t], pdf(x)*exp(I*x*t), 'mpmath')
-        cf2 = lambda t: mpmath.nsum(lambda x: f(x, t), [support_lower_limit, support_upper_limit], maxdegree=10)
+        def cf2(t): return mpmath.nsum(lambda x: f(x, t), [
```

Comment:
No need to modify this. It is fine that you split the statement.
```suggestion
        cf2 = lambda t: mpmath.nsum(lambda x: f(x, t), [
```